src/app/components/FormElements/DateTimeField.tsx

```diff
@@ -4,12 +4,26 @@ import { default as FormFieldWrapper, FormFieldWrapperProps } from './FormFieldW
 import * as DateTime from 'react-datetime';
 import 'react-datetime/css/react-datetime.css';
 
-export default class DateTimeField extends React.PureComponent<FormFieldWrapperProps>{
+export interface DateTimeFieldProps{
+    showDate?: boolean;
+    showTime?: boolean;
+    closeOnSelect?: boolean;
+}
+
+export default class DateTimeField extends React.PureComponent<FormFieldWrapperProps & DateTimeFieldProps>{
+    onChange(ev:any){
+        let newValue = ev;
+        if(moment.isMoment(ev)){
+            newValue = ev.toISOString();
+        }
+        this.props.input.onChange(newValue);
+    }
+
     render(){
-        const {input:{value, onChange}} = this.props;
+        const {input:{value}, showDate, showTime, closeOnSelect} = this.props;
```

Comment:
Maybe some defaults for the new properties?

It seems that `closeOnSelect` is generally used as false and maybe the other show properties as true?